### ISSUE TITLE
[MsTeams][Desktop/Mobile][Carousel taking time to load on the conversation window of MS Teams]

### ISSUE THREAD

**Issue by Sujit-Gaikwad** [OfficeDev/BotBuilder-MicrosoftTeams#237]
HI ,

I am facing issue with card Carousel. this is  taking time to load on the conversation window of MS Teams.

can you please look into issue??

regards,
Sujit

**Comment by Subhasish-msft** [comment]
How long does the card takes to render ? Also can you share the card json payload so that we can reproduce it

**Comment by Sujit-Gaikwad** [comment]
@Subhasish-msft sorry, I forgot to upload details at  time of raising issue 

It is taking time between 5- 10 sec approx. please find attached adaptive cards  json that we have used  in carousal.

[CardCarousalAttachment.docx](https://github.com/OfficeDev/BotBuilder-MicrosoftTeams/files/4540326/CardCarousalAttachment.docx)

**Comment by Sujit-Gaikwad** [comment]
@Subhasish-msft same card I have tested on iOS device but it is taking more time than other platform 
and text within card is not getting visible as well.

if we refresh the chat window the card is not rendering properly.
please find the attachment :

iPhone View:
![iPhone Device](https://user-images.githubusercontent.com/60354515/80496014-438a9500-8986-11ea-9168-142814fbf4df.png)


Android view:

![Android Phone](https://user-images.githubusercontent.com/60354515/80496522-f0fda880-8986-11ea-8ae9-791359c4ea39.jpg)

**Comment by Subhasish-msft** [comment]
@Sujit-Gaikwad , We have created a bug for this. Can you please share the iOS build version and teams version.

**Comment by Sujit-Gaikwad** [comment]
IPhone 6
iOS version : 12.4.5
Teams version : 2.0.10

**Comment by Sujit-Gaikwad** [comment]
HI @Subhasish-msft,

I want to add one more scenario on this issue. 
-> First I triggered a carousel from bot
-> after getting an adaptive card carousel consist of buttons in the conversation pane.
-> I clicked on the button on the card I get the result.

after this, if I scroll back to select another action from the carousel but I'm getting 
white screen area like :

![Carosal](https://user-images.githubusercontent.com/60354515/83634151-aee80800-a5bf-11ea-8693-ce1b6191241f.JPG)
